### ISSUE TITLE
Moving files results in lower case nav titles

### ISSUE THREAD

**Issue by thenktor** [typemill/typemill#269]
Hi,

when doing something like this on command line:
```
# mv 02-groupware/03-exchange/ .
```
the nav titles of the 03-exchange folder and articles below change to lower case:

![ksnip_20210708-142539](https://user-images.githubusercontent.com/371714/124921150-6556c180-dff8-11eb-8be7-2c2bbf45cc17.png)

Therefore you have to edit the nav titles on the meta tab to get it back to normal.

**Comment by trendschau** [comment]
Yes, I can't guarantee that everything works as expected if you use the file system directly (or an external text-editor) instead of using the author interface. It is probably because the events are missing that update the cache files with additional information like navigation title or hide/not hide. If those information are missing, then it falls back to the titles that are generated from the file-names. You can try to delete the cache files (delete manually or use the button in the admin interface). Does that help?

**Comment by thenktor** [comment]
Sorry, forgot to mention: I've already tried to delete the content of the cache dir. Doesn't make a difference.

**Comment by trendschau** [comment]
Sorry for delay: I just tried it with a live website and ftp, the workflow is the following: 

* Rename a folder with console or ftp
* delete the cache in the settings
* go to the frontend website and do a hard refresh (or wait 10 minutes).
* Then the navigation in frontend and backend is recreated automatically and the navigation names from meta-settings are there again. 

The workflow is not very intuitive, I know, but the use case is not very common, so I do not really want to spend time to improve it. One improvement could be to change the button function from "clear cache" to "recreate cache" but for me it is low prio right now. 

Does that help?

**Comment by thenktor** [comment]
Indeed it works that way.

**Comment by trendschau** [comment]
I changed the clear cache functionality to a recreate cache functionality, so the important parts of the cache will work without visiting a frontend page as of version 1.4.9.
